src/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+
+export default function Header() {
+    return (
+        <View style={styles.headerContainer}>
+            <Text style={styles.headerTitle}>Todo App</Text>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    headerContainer: {
```

Comment:
We can go ahead and drop `header` here and just call this `container`

Reply:
ok. will update that.